src/hooks/useLogDownloader/index.ts

```diff
@@ -34,6 +42,21 @@ const useLogDownloader = (url: string, logType: LogTypes) => {
       onProgress: (progress) => {
         setFileSize(getFileSize() + progress);
       },
+      downloadSizeLimit,
+      onIncompleteDownload: (reason, incompleteDownloadError) => {
+        leaveBreadcrumb(
+          "useLogDownloader",
+          { incompleteDownloadError, reason },
+          "error"
+        );
+        dispatchToast.warning(
+          "We were only able to partially download this log. Use the Evergreen CLI command in the details menu to download the log on to your machine.",
```

Comment:
This isn't included as part of the diff for this pr. (See description) But this statement would not be true since local downloads are also susceptible to timeouts.